File: Level 4 Batch 10/CPP/Introduction/type_casting.cpp
#include<iostream>
#include<string>
using namespace std;

int main() {
    int a = 100;
    float b = 2.5;

    int c = (int)b; 
    //explicit type conversion
    cout << "b is " << b << endl;
    cout << "c is "<< c << endl;

    float d = a; 
    //implicit type conversion
    cout << "a is "<< a << endl;
    cout << "d is "<< d << endl;

    return 0;
}

File: Level 3 Batch 4/Java/Daily/01-Introduction/src/com/mmit/day2/TypeCasting.java
package com.mmit.day2;

public class TypeCasting {	

	public static void main(String[] args) {
		
		int a = 100;
		long b = a ; //implicit casting
		double c = a;
		
		System.out.println("a: " + a);
		System.out.println("b: " + b);
		System.out.println("c: " + c);
		
		System.out.println("----------------");
		
		double d = 3.3;
		int e = (int)d; //explicit casting
		
		System.out.println("d: "+ d);
		System.out.println("e: " + e);
	}

}
